src/selfplay/loop.cc

```diff
@@ -123,13 +126,46 @@ void SelfPlayLoop::CmdSetOption(const std::string& name,
 }
 
 void SelfPlayLoop::SendTournament(const TournamentInfo& info) {
+  int winp1   = info.results[0][0] + info.results[0][1];
```

Comment:
I vaguely remember that I already reviewed that function somewhere..
Names are too cryptic. `player1_wins` ? consider adding `const` for all variables that don't change.

Reply:
Yes this was supposed to add on top of another PR that provided stats for self-play matches. But unfortunately that author vanished. As i needed it testing CP, i "sneaked" it in here. Will add conts.